pkg/controller/machine.go

```diff
@@ -433,7 +491,7 @@ func (c *controller) machineCreate(machine *v1alpha1.Machine, driver driver.Driv
 			LastUpdateTime: metav1.Now(),
 		}
 		currentStatus := v1alpha1.CurrentStatus{
-			Phase:          v1alpha1.MachineFailed,
+			Phase:          v1alpha1.MachineCrashLoopBackOff,
```

Comment:
This is while deletion right? Should this be `MachineCrashLoopBackOff` or `MachineTerminating`? as otherwise the deletion starts switching the conditions between the two?

Reply:
This is a machineCreate method, right.

Reply:
Yes, sorry I got confused.